sklearn/discriminant_analysis.py

```diff
@@ -534,7 +546,13 @@ def _solve_svd(self, X, y):
         # Centers are living in a space with n_classes-1 dim (maximum)
         # Use SVD to find projection in the space spanned by the
         # (n_classes) centers
-        _, S, Vt = svd(X, full_matrices=False)
+        import dask.array as da
+
+        if isinstance(X, da.Array):
+            # dask can flip signs of Vt sometimes
+            _, S, Vt = svd(X, full_matrices=False, coerce_signs=False)
```

Comment:
Not sure if the sign should matter in the result.

If it does, I can fix the svd inside array-api-compat to do ``coerce_signs=False``